pq-crypto/pq-random.c

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+#include <sys/param.h>
+
+#include "pq-random.h"
```

Comment:
We try to always include the path in our includes, e.g. #include "pq-crypto/pq-random.h" , it just makes the context more clear, and usually simplifies naming too. 

We also try to put a blank line between the includes from each directory.